### PR TITLE
[8.0] Record rules: show in AccessError message also the user who is trying to access and the record they are trying to access to.

### PR DESCRIPTION
**Pull request by eLBati**
See https://github.com/odoo/odoo/pull/13030

### CODE REVIEW

File: openerp/models.py [hunk 1]
```diff
@@ -3394,8 +3394,8 @@ def qualify(field):
                     ))
             # store an access error exception in existing records
             exc = AccessError(
-                _('The requested operation cannot be completed due to security restrictions. Please contact your system administrator.\n\n(Document type: %s, Operation: %s)') % \
-                (self._name, 'read')
+                _('The requested operation cannot be completed due to security restrictions. Please contact your system administrator.\n\n(Document type: %s, Operation: %s, Records: %s, User: %s)') % \
+                (self._name, 'read', ','.join([str(r.id) for r in self]), self._uid)
```
Comment (dreispt) [inline openerp/models.py]: Records could be a large list. Maybe it's better limit this to, say, the first 6 records?

Comment (eLBati) [inline openerp/models.py]: @dreispt could you make an example where records would be a large list?

Comment (dreispt) [inline openerp/models.py]: Any write operation affecting a large number of records.

**Comment by eLBati** [comment]
@dreispt done

**Comment by mart-e** [comment]
This is an improvement so should target at least the version 10 or master.
Closing, let's keep the discussion on #13030 and see where to merge later

**Pull request closed without merging**